docs/static/filebeat_modules/apache2/access/pipeline.conf

```diff
@@ -0,0 +1,36 @@
+input {
```

Comment:
@suyograo Some of the example Apache2 log files from the Beats directory cause Logstash to crash when I use this config.

The config works when I use it to parse this log: https://github.com/elastic/beats/blob/master/filebeat/module/apache2/access/test/ubuntu-2.2.22.log

But when I try to use it to parse either of the following logs, I see errors: 
https://github.com/elastic/beats/blob/master/filebeat/module/apache2/access/test/darwin-2.4.23.log and https://github.com/elastic/beats/blob/master/filebeat/module/apache2/access/test/test.log

The Apache2 Filebeat Module is able to parse all of these logs.

Note that I have modified the LS config slightly from the example by adding the following lines to the config for the ES output so that I can use the filebeat index:

`````
    manage_template => false
    index => "%{[@metadata][beat]}-%{+YYYY.MM.dd}"
    document_type => "%{[@metadata][type]}"
`````

Here's my Filebeat config:

`````
filebeat.prospectors:
- input_type: log
  paths:
    - /Users/dedemorton/BuildTesting/5.3.0_e5a8c674/filebeat-5.3.0-darwin-x86_64/log_examples/apache_access/darwin-2.4.23.log*
  exclude_files: [".gz$"]
output.logstash:
  hosts: ["localhost:5044"]

`````

Here are the errors that I see when the log file is `darwin-2.4.23.log` or `test.log`:

`````

[2017-03-21T10:47:10,780][INFO ][logstash.agent           ] Successfully started Logstash API endpoint {:port=>9600}
[2017-03-21T10:47:19,552][FATAL][logstash.runner          ] An unexpected error occurred! {:error=>#<LogStash::Error: timestamp field is missing>, :backtrace=>["org/logstash/ext/JrubyEventExtLibrary.java:202:in `sprintf'", "/Users/dedemorton/BuildTesting/5.3.0_e5a8c674/logstash-5.3.0/vendor/bundle/jruby/1.9/gems/logstash-output-elasticsearch-6.2.6-java/lib/logstash/outputs/elasticsearch/common.rb:153:in `event_action_params'", "/Users/dedemorton/BuildTesting/5.3.0_e5a8c674/logstash-5.3.0/vendor/bundle/jruby/1.9/gems/logstash-output-elasticsearch-6.2.6-java/lib/logstash/outputs/elasticsearch/common.rb:40:in `event_action_tuple'", "/Users/dedemorton/BuildTesting/5.3.0_e5a8c674/logstash-5.3.0/vendor/bundle/jruby/1.9/gems/logstash-output-elasticsearch-6.2.6-java/lib/logstash/outputs/elasticsearch/common.rb:34:in `multi_receive'", "org/jruby/RubyArray.java:2414:in `map'", "/Users/dedemorton/BuildTesting/5.3.0_e5a8c674/logstash-5.3.0/vendor/bundle/jruby/1.9/gems/logstash-output-elasticsearch-6.2.6-java/lib/logstash/outputs/elasticsearch/common.rb:34:in `multi_receive'", "/Users/dedemorton/BuildTesting/5.3.0_e5a8c674/logstash-5.3.0/logstash-core/lib/logstash/output_delegator_strategies/shared.rb:12:in `multi_receive'", "/Users/dedemorton/BuildTesting/5.3.0_e5a8c674/logstash-5.3.0/logstash-core/lib/logstash/output_delegator.rb:47:in `multi_receive'", "/Users/dedemorton/BuildTesting/5.3.0_e5a8c674/logstash-5.3.0/logstash-core/lib/logstash/pipeline.rb:390:in `output_batch'", "org/jruby/RubyHash.java:1342:in `each'", "/Users/dedemorton/BuildTesting/5.3.0_e5a8c674/logstash-5.3.0/logstash-core/lib/logstash/pipeline.rb:389:in `output_batch'", "/Users/dedemorton/BuildTesting/5.3.0_e5a8c674/logstash-5.3.0/logstash-core/lib/logstash/pipeline.rb:346:in `worker_loop'", "/Users/dedemorton/BuildTesting/5.3.0_e5a8c674/logstash-5.3.0/logstash-core/lib/logstash/pipeline.rb:306:in `start_workers'"]}
`````